Bottom: 034811e9d70bf4fca1ef025a9e266940f41826cb
Top:    9dae5c54cda195cd3d3fe6d457f4b286139c7509
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-06 12:54:20 +0800

Refresh of autofs-5.0.5-fix-master-map-bogus-keywork-match.patch

---

diff --git a/lib/master_tok.l b/lib/master_tok.l
index aec857e..df56be0 100644
--- a/lib/master_tok.l
+++ b/lib/master_tok.l
@@ -92,6 +92,7 @@ NL		\r?\n
 CONT		\\\n{OPTWS}
 
 OPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(=([[:alnum:]_\-])+)*)+)
+QVOPTIONSTR	([\-]?([[:alpha:]_]([[:alnum:]_\-])*(="([[:alnum:]_\-\:])+")))
 MACROSTR	(-D{OPTWS}([[:alpha:]_]([[:alnum:]_\-\.])*)=([[:alnum:]_\-\.])+)
 SLASHIFYSTR	(--(no-)?slashify-colons)
 NUMBER		[0-9]+
@@ -400,6 +401,11 @@ OPTNTOUT	(-n{OPTWS}|-n{OPTWS}={OPTWS}|--negative-timeout{OPTWS}|--negative-timeo
 		return(OPTION);
 	}
 
+	{QVOPTIONSTR} {
+		strcpy(master_lval.strtype, master_text);
+		return(OPTION);
+	}
+
 	{OPTIONSTR} {
 		strcpy(master_lval.strtype, master_text);
 		return(OPTION);
